clone_repos.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+MIRROR="https://github.com"
+function gc () {
+git clone $MIRROR/$2 -b $3 $1
+pip install $1
```

Comment:
upper-constraints?

Reply:
Well, as we move downstream, we can't rely upon that. I think we need to generate a list of packages/versions and shift that over. such that we run with --no-deps and that way also things should implode if we didn't get everything.